content/en/docs/configuring-sdk/_index.md

```diff
@@ -67,19 +67,259 @@ export AWS_REGION=us-west-2
+operating system. In Windows, you can refer to your home directory by
+using the environment variable :code:`%UserProfile%`. In Unix-like systems, you
+can use the environment variable :code:`$HOME` or :code:`~` (tilde).
+
+If you already use this file for other SDKs and tools (like the {{% alias tools=CLI %}}),
+you don't need to change anything to use the files in this SDK. If
+you use different credentials for different tools or applications, you
+can use *profiles* to configure multiple access keys in the same
+configuration file.
+
+#### Creating the Credentials File
+
+If you don't have a shared credentials file (`.aws/credentials`), you
+can use any text editor to create one in your home directory. Add the
+following content to your credentials file, replacing
+`<YOUR_ACCESS_KEY_ID>` and `<YOUR_SECRET_ACCESS_KEY>` with your
+credentials.
+
+```ini
+[default]
+aws_access_key_id = <YOUR_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_SECRET_ACCESS_KEY>
+```
+
+The `[default]` heading defines credentials for the default profile,
+which the SDK will use unless you configure it to use another profile.
+
+You can also use temporary security credentials by adding the session
+tokens to your profile, as shown in the following example:
+
+```ini
+[temp]
+aws_access_key_id = <YOUR_TEMP_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_TEMP_SECRET_ACCESS_KEY>
+aws_session_token = <YOUR_SESSION_TOKEN>
+```
+
+#### Specifying Profiles
+
+You can include multiple access keys in the same configuration file by
+associating each set of access keys with a profile. For example, in your
+credentials file, you can declare multiple profiles, as follows.
+
+```ini
+[default]
+aws_access_key_id = <YOUR_DEFAULT_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_DEFAULT_SECRET_ACCESS_KEY>
+
+[test-account]
+aws_access_key_id = <YOUR_TEST_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_TEST_SECRET_ACCESS_KEY>
+
+[prod-account]
+; work profile
+aws_access_key_id = <YOUR_PROD_ACCESS_KEY_ID>
+aws_secret_access_key = <YOUR_PROD_SECRET_ACCESS_KEY>
+```
+
+By default, the SDK checks the `AWS_PROFILE` environment variable to
+determine which profile to use. If no `AWS_PROFILE` variable is set,
+the SDK uses the default profile.
+
+If you have an application named `myapp` that uses the SDK, you can
+run it with the test credentials by setting the variable to
+`test-account myapp`, as shown in the following command.
+
+```bash
+$ AWS_PROFILE=test-account myapp
+```
+
+You can also use instruct the SDK to select a profile by either
+`os.Setenv("AWS_PROFILE", "test-account")` before calling `config.LoadDefaultConfig`,
+or by passing an explicit profile as an argument as shown in the following example:
+
+```go
+cfg, err := config.LoadDefaultConfig(config.WithSharedConfigProfile("test-account"))
+```
+
+In addition, checking if your credentials have been found is fairly easy.
+
+```go
+_, err := sess.Config.Credentials.Get()
```

Comment:
Forgot to update this, though I would argue this should be removed, as this isn't necesserily a valid test given that `AnonymousCredentials` would be set if no credentials were found and this would not return an error.